### ISSUE TITLE
Not syncing. Missing some sync item selections?

### ISSUE THREAD

**Issue by rakheshster** [mozilla/multi-account-containers#1687]
Hi,

When the latest version of MAC got installed it asked if I want to enable syncing. I did so but none of my containers are syncing. Under items to sync in Firefox I am only syncing Bookmarks, History, Open tabs, and Logins ... so could the fact that I am not syncing add-ons or preferences be the reason? 

I am using the latest version of Firefox and Firefox Developer and the issue is the same on both. Using macOS. 

Thanks.

**Comment by k4n30** [comment]
> so could the fact that I am not syncing add-ons or preferences be the reason?

Yes - There's no point "syncing" containers if you're not syncing addons (of which containers is one of)

**Comment by rakheshster** [comment]
Cool, thanks for that. I use Firefox at home and work and don't want my work addons to sync over home, so had addons sync disabled. I have MAC installed on both machines though so that'd be enough to have them sync. Suppose one wanted to manually keep containers in sync, is there a way? I know there's the containers.json file, but that's just got the container info I guess and not the domain to container mapping.

**Comment by hsnali** [comment]
Just to chime in here, I have FF Developer edition, signed in, have addon-sync enabled, set up my containers. I wanted to use regular FF on another machine and turned on sync, all my addons and other settings come over, but containers are empty, only the defaults.

Are we meant to be able to sync between two different builds of Firefox such as Developer and regular?

**Comment by jasonemann** [comment]
I've just reinstalled my Fedora system and have updated Firefox to 74.0.  I have a Firefox account with sync enabled for preferences, addons and bookmarks.  After signing into my Firefox account, the Multi Account Containers was automatically installed along with all my other addons, but none of its settings have synced.  Enable Sync is checked in the addon's preferences, as it was on my previous installation, but none of the containers I previously created have reappeared, and the "Always open in {container}" preference for various sites are lost.

**Comment by rakheshster** [comment]
Any suggestions on what files to copy over if one wanted to keep the container info on two Firefox installs in sync?

**Comment by serovar** [comment]
> I've just reinstalled my Fedora system and have updated Firefox to 74.0. I have a Firefox account with sync enabled for preferences, addons and bookmarks. After signing into my Firefox account, the Multi Account Containers was automatically installed along with all my other addons, but none of its settings have synced. Enable Sync is checked in the addon's preferences, as it was on my previous installation, but none of the containers I previously created have reappeared, and the "Always open in {container}" preference for various sites are lost.

Exact same situation here with Firefox Developer Edition 76.0b3 on a Windows 10 and a Fedora 31 machine.

**Comment by onecrayon** [comment]
Same problem for me in Firefox 78.0.2 on macOS and Windows. I setup MAC on macOS and it was working great. Switched to Windows, enabled "Sync Add-ons", and MAC was automatically installed, but only with defaults (all settings from macOS are missing). Have had no luck getting it to sync at all, which is really disappointing.

**Comment by dannycolin** [comment]
> [I] had addons sync disabled. I have MAC installed on both machines though so that'd be enough to have them sync. 

AFAIK, you need addons to be synced if you want addons data to also be synced. I need to do some test to confirm that tho.

> Suppose one wanted to manually keep containers in sync, is there a way? 

Currently, there's no way to export your container data.
